### ISSUE TITLE
Device c8y_Unavailability triggers on newly provisioned device

### ISSUE THREAD

**Issue by logan-jt** [thin-edge/thin-edge.io#2963]
**Describe the bug**

Thin Edge Device "Send connection" status appears offline shortly after provisioning. "c8y_UnavailabilityAlarm" Present

**To Reproduce**

Re-provision a new Raspberry pi using standard process outlined in the "A tour of thin-edge.io" documentation. Once the cert has been uploaded and c8y connected, the device connects correctly. Approx 5 mins after initially connecting, the c8y_UnavailabilityAlarm activates and the device status appears offline. 

**Environment (please complete the following information):**
 - OS [incl. version]
PRETTY_NAME="Raspbian GNU/Linux 11 (bullseye)"
NAME="Raspbian GNU/Linux"
VERSION_ID="11"
VERSION="11 (bullseye)"
VERSION_CODENAME=bullseye
ID=raspbian
ID_LIKE=debian

 - Hardware [incl. revision]
Raspberry Pi 4b
 - System-Architecture [e.g. result of "uname -a"]
Linux raspberrypi 6.1.21-v8+ #1642 SMP PREEMPT Mon Apr  3 17:24:16 BST 2023 aarch64 GNU/Linux
 - thin-edge.io version [e.g. 0.1.0]
1.1.1
**Additional context**
Issue discussed in Thin Edge Discord HELPPOINT channel
[tedge-mapper-c8y logs.txt](https://github.com/user-attachments/files/15982431/tedge-mapper-c8y.logs.txt)
[Mosquitto_Log.txt](https://github.com/user-attachments/files/15982432/Mosquitto_Log.txt)

**Comment by reubenmiller** [comment]
@logan-jt Thank for the ticket.

TLDR; We will be merging a [Pull Request](https://github.com/thin-edge/thin-edge.io/pull/2940) soon which includes out-of-the-box support for the Cumulocity IoT Availability monitoring feature.

**Background**

The availability alarm is set by the platform when it detects that no data (measurements/events/alarms and inventory (e.g. `/twin/+` tedge topic)) has be received from the device within the defined "required interval". When the value is set from the UI, only 

The default configuration of thin-edge.io is to not send any data back to the backend, so by default there will be no data sent by thin-edge.io to reset the platform's calculation. From the getting started guide, if you configured the **collectd** settings, then **collectd** would be one source of these measurements, however the default interval configured for most of the measurements in in the realm of 10-15 minutes, so in that case you'd see the availability go between AVAILABLE and UNAVAILABLE (at seemingly odd intervals due to the misalignment of the required interval and data sent from the device).

Moving forward, once [PR2940](https://github.com/thin-edge/thin-edge.io/pull/2940) is merged, it will send a heartbeat signal (an empty json object via the inventory topic) to Cumulocity IoT based on the status of service (default being the `tedge-agent`). The feature means that you won't have to send any other additional telemetry data to the cloud in order to indicate that the device is actively communicating.

**Comment by reubenmiller** [comment]
Closing as the required interval should be set to some multiple of the minimum interval at which telemetry data is sent from the device, but the Cumulocity IoT Availability monitor will be supported out of the box (using heartbeat signals), once https://github.com/thin-edge/thin-edge.io/pull/2940 is merged
